<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>VChooseModeDialog</class>
 <widget class="QDialog" name="VChooseModeDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>201</width>
    <height>137</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Choose mode</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <widget class="QGroupBox" name="m_groupBox">
     <property name="minimumSize">
      <size>
       <width>180</width>
       <height>90</height>
      </size>
     </property>
     <property name="title">
      <string>Calibration mode</string>
     </property>
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>30</y>
        <width>123</width>
        <height>48</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QRadioButton" name="m_stereoRadio">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>Stereo calibration</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="m_monoRadio">
         <property name="text">
          <string>Mono calibration</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>3</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="m_okButton">
       <property name="text">
        <string>OK</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="m_cancelButton">
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <tabstops>
  <tabstop>m_stereoRadio</tabstop>
  <tabstop>m_monoRadio</tabstop>
  <tabstop>m_okButton</tabstop>
  <tabstop>m_cancelButton</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>m_okButton</sender>
   <signal>clicked()</signal>
   <receiver>VChooseModeDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>79</x>
     <y>140</y>
    </hint>
    <hint type="destinationlabel">
     <x>77</x>
     <y>154</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>m_cancelButton</sender>
   <signal>clicked()</signal>
   <receiver>VChooseModeDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>122</x>
     <y>141</y>
    </hint>
    <hint type="destinationlabel">
     <x>142</x>
     <y>113</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
